src/folsom_webmachine_metrics_resource.erl

```diff
@@ -90,10 +90,11 @@ resource_exists(Id, ReqData, Context) ->
 get_request(undefined, undefined, undefined) ->
     folsom_metrics:get_metrics();
 get_request(Id, undefined, undefined) ->
-    case folsom_metrics:get_metric_info(Id) of
-        [{_, [{type, histogram}]}] ->
+    MetricInfo = folsom_metrics:get_metric_info(Id),
```

Comment:
MetricInfo can look something like this (this is what triggered the issue for me):
```
[{connect_latency,[{type,histogram},
                   {tags,{set,0,16,16,8,80,48,
                              {[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],
                               []},
                              {{[],[],[],[],[],[],[],[],[],[],[],[],[],[],[],
                                []}}}}]}]
```